posts/2017-01-10-dat-desktop.md

```diff
@@ -0,0 +1,41 @@
+# Dat Desktop Release
+
+Today we're announcing the release of Dat Desktop, a peer to peer data sharing
+app built for humans.
+
+Dat Desktop is a new way to share data with friends & colleagues. Using the
+peer to peer (p2p) Hyperdrive protocol, it makes sharing folders on your
+computer secure and convenient. No centralized servers needed.
+
+## Installing Dat Desktop
+
+You can [download Dat Desktop here][download] and read [the source code
+here][source]. Found an issue, got a question? [Open an issue on GitHub][gh] or
+reach us [on Twitter][twitter].
+
+## Hyperdrive protocol
+
+The Hyperdrive protocol is a protocol for p2p directories. It was designed to
+efficiently share terabytes of of scientific data in real time between research
+institutions on multiple continents, but it works fine to share text files on
+the local network too
+
+## Updating Dat Desktop
+
+You only have to install Dat Desktop once - we'll make sure you'll always be
+running the latest and greatest. Some of the features to come include:
+
+- preview modes for a wide range of data formats
+- tagging, versioning and shapshotting of Archives
+- publishing Archives to Dat Cloud (coming soon™)
```

Comment:
The registry associates a Dat link with a **username/dataset** (so you can do things like `dat clone username/dataset) but it does not host/backup any data. Right now we'll assume all links published to the registry are public.

The "cloud" will actually *host* data from archives. The data could be private or public. But it will essentially provide constant uptime for the Dats you put on the cloud (like our archiver-bot) and provide backup.